packages/components/src/utils/getBreadcrumbFromSiteMap.ts

```diff
@@ -11,7 +11,12 @@ export const getBreadcrumbFromSiteMap = (
   let node = sitemap
   let currentPath = ""
 
-  for (const pathSegment of permalink) {
+  breadcrumb.push({
+    title: "Home",
+    url: "/",
+  })
```

Comment:
could we do 
```ts
const breadcrumb = [{
    title: "Home",
    url: "/",
  }]
```

looks slightly neater, benefits are that we don't directly mutate the `breadcrumb`

Reply:
arhhh thank you! addressed in [632c4ae](https://github.com/opengovsg/isomer/pull/125/commits/632c4ae8d14136ec3859bd3d5b82237629ff20e3)